app/src/main/res/layout/review_row.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:id="@+id/review_preview_row"
+              android:layout_width="match_parent"
+              android:layout_height="@dimen/album_art_size"
+              android:orientation="horizontal"
+              android:weightSum="1"
+              android:background="@drawable/album_border"
+              android:padding="@dimen/album_art_border">
+
+    <ImageView
+        android:id="@+id/imageview_album_art"
+        android:layout_height="match_parent"
+        android:layout_width="@dimen/album_art_size"
+        android:src="@mipmap/ic_launcher"/>
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:background="@drawable/album_border"
+        android:padding="@dimen/album_info_border">
+
+        <RelativeLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_weight="0.5">
+            <TextView
+                android:id="@+id/textview_album_name"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_weight="0.5"
+                android:gravity="center"
+                android:text="@string/placeholder_album_name"
+                android:textColor="@android:color/black"
+                android:textSize="@dimen/list_font_size"
+                android:textStyle="bold"/>
+
+            <Button
+                android:id="@+id/button_delete"
+                android:layout_width="40dp"
```

Comment:
because it covers the name if it is (the width goes really wide for just an X :/), but i will put these numbers and text from this into strings etc.